docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+    # rif-relay:
```

Comment:
Erase rif-relay service if its not used.

Reply:
doen